S100-Shop-Facade-Open-Api-Sepc-v0.1.yaml

```
@@ -0,0 +1,1145 @@
+openapi: 3.0.1
+info:
+  title: UKHO S100 Shop Facade Service APIs
+  description: |
+    The UKHO S100 Shop Facade Service APIs.
+  contact:
+    name: UKHO
+    email: contact@ukho.co.uk
+  version: 0.0.1
+servers:
+  - url: https://shop-facade-service-url
+    description: Proposed Dev URL
+security:
+  - jwtBearerAuth: []
+  
+paths:
+  /permits/S100/{licenceId}:
```

We should try and be consistent in the paths and have either {licenceId} first, or s100 first, when compared to the `getUserPermits` operation.  My feeling is that both should have the licenceId first.

```suggestion
  /permits/{licenceId}/S100:
```

---

Updated.